src/transaction/log_impl.h

```diff
@@ -838,7 +843,7 @@ extern int logpb_rename_all_volumes_files (THREAD_ENTRY * thread_p, VOLID num_pe
 extern int logpb_delete (THREAD_ENTRY * thread_p, VOLID num_perm_vols, const char *db_fullname, const char *logpath,
 			 const char *prefix_logname, bool force_delete);
 extern int logpb_check_exist_any_volumes (THREAD_ENTRY * thread_p, const char *db_fullname, const char *logpath,
-					  const char *prefix_logname, char *first_vol, bool * is_exist);
+					  const char *prefix_logname, char *first_vol, bool *is_exist);
```

Comment:
There is an imperfection of the indent system.

Before introducing C++ and astyle, the rules for spaces between * and names was this:

  - LOG_LSA * lsa;  // non-primitive types: space to the left and right of *
  - int *lsa; // primitive types: space only to the left
I don't know the reason indent treats primitives and non-primitives differently.

Astyle can't do spaces both to the left and to the right. So between having a space to the left or space to the right, we chose only to the left, because that's at least how indent handled primitive case.

So having space between */& and name, or having no space between them are both acceptable options. Just go with whichever you prefer and let indent/astyle handle the rest :).